packages/app/src/components/DBTraceWaterfallChart.tsx

```diff
@@ -580,6 +592,7 @@ export function DBTraceWaterfallChartContainer({
           initialScrollRowIndex={initialScrollRowIndex}
         />
       )}
-    </>
+      <div className={resizeStyles.resizeYHandle} onMouseDown={startResize} />
```

Comment:
Also a small thing, but I had trouble finding this handle initially - it seems odd to me that the handle is just an empty div and not the <Divider> line right below the waterfall.

Reply:
Good idea!